cmd/simulator/load/loader.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package load
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+	"os"
+
+	"github.com/ava-labs/subnet-evm/cmd/simulator/config"
+	"github.com/ava-labs/subnet-evm/cmd/simulator/key"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func CreateLoader(ctx context.Context, config config.Config) (*WorkerGroup, error) {
+	// Construct the arguments for the load simulator
+	switch {
+	case len(config.Endpoints) == 0:
+		fmt.Printf("Must specify at least one clientURI\n")
```

Comment:
Can we not validate the config before we get to this point?

Reply:
Good call